templates/mongodb.conf.2.6.erb

```diff
@@ -129,6 +129,9 @@ replication.replSetName: <%= @replset %>
 <% if @oplog_size -%>
 replication.oplogSizeMB: <%= @oplog_size %>
 <% end -%>
+<% if defined?(@repl_enable_majority_read_concern) -%>
```

Comment:
I think this will always be defined. Or does it return false when its value is `undef`?

Reply:
Since the default is true, I could have just tested for == false, but in the event that it is not defined, I didn't want the entry to show up in the config file at all.